Site/webapp/wdkCustomization/js/client/components/Srt.tsx

```diff
@@ -225,19 +309,27 @@ function useCompatibleSrtFormConfigs() {
     [ recordClasses ]
   );
 
-  const compatibleSrtConfigs = useMemo(() =>
-    recordClassUrlSegments != null &&
-    projectId != null &&
-    srtQuestionParamDisplayMap != null &&
-    SUPPORTED_RECORD_CLASS_CONFIGS
-      .filter(initialSrtConfig => recordClassUrlSegments.has(initialSrtConfig.recordClassUrlSegment))
-      .map(initialSrtConfig => ({
-        ...initialSrtConfig,
-        initialIdsState: initialSrtConfig.makeInitialIdsState(srtQuestionParamDisplayMap),
-        projectId
-      }) as SrtFormConfig),
-    [ recordClassUrlSegments, projectId, srtQuestionParamDisplayMap ]
+  const compatibleSrtConfigs = useMemo(() => {
+    if (
+      recordClassUrlSegments != null &&
+      projectId != null &&
+      srtQuestionParamDisplayMap != null &&
+      storedSrtData.value != null
+    ) {
+      return storedSrtData.value
+          .filter((initialSrtConfig) => recordClassUrlSegments.has(initialSrtConfig.recordClassUrlSegment))
+          .map((initialSrtConfig): SrtFormConfig => ({
+            ...initialSrtConfig,
+            defaultIdsState: initialSrtConfig.makeDefaultIdsState(srtQuestionParamDisplayMap),
+            initialIdsState: initialSrtConfig.makeInitialIdsState(srtQuestionParamDisplayMap),
```

Comment:
```suggestion
            initialIdsState: initialSrtConfig.storedIdsState ?? initialSrtConfig.makeInitialIdsState(srtQuestionParamDisplayMap),
```

(And similarly for `initialReporterFormState`.)